src/main/java/com/codecool/virtualstylist/wardrobe/WardrobeService.java

```diff
@@ -16,9 +16,18 @@ public WardrobeService(@Qualifier("wardrobeDataAccess") WardrobeDataAccess wardr
         this.wardrobeDataAccess = wardrobeDataAccess;
     }
 
+    void addCloth(Cloth cloth){
+        wardrobeDataAccess.save(cloth);
+    }
+
+    void editCloth(Cloth cloth){
+        wardrobeDataAccess.save(cloth);
```

Comment:
Hibernate has '.saveOrUpdate(entity)' method . Maybe we should use that one instead? I'm not sure whether   it will work as we will or not. I think we will have to load entity we want to change, set new properties and save back? 🤔